.github/workflows/ci.yml

```diff
@@ -147,7 +147,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-11, windows-2019]
+        os: [windows-2019]
```

Comment:
Whole matrix can be dropped now, instead of:
```
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        os: [windows-2019]
```

you should be able to go with:
```
    runs-on: windows-2019
```

Reply:
Btw – shouldn't we consider `windows-latest` in here? Currently it'll be `windows-2022`.

Reply:
perhaps I should leave the matrix in and test on two windows versions?